dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/ApplicationBuilder.cs

```diff
@@ -9,15 +9,13 @@ namespace Microsoft.Teams.AI
     /// A builder class for simplifying the creation of an Application instance.
     /// </summary>
     /// <typeparam name="TState">Optional. Type of the turn state. This allows for strongly typed access to the turn state.</typeparam>
-    /// <typeparam name="TTurnStateManager">Type of the turn state manager.</typeparam>
-    public class ApplicationBuilder<TState, TTurnStateManager>
-        where TState : ITurnState<StateBase, StateBase, TempState>
-        where TTurnStateManager : ITurnStateManager<TState>, new()
+    public class ApplicationBuilder<TState>
+        where TState : TurnState<Record, Record, TempState>, new()
```

Comment:
Should be `ITurnState`

Reply:
And should add the _load_ and _save_ methods to the interface so that they can be called from the application class.